Makefile

```diff
@@ -384,22 +376,31 @@ $(LANGUAGES_MO_FILES): $(LANGUAGES_PO_FILES)
 	${PYTHON} setup.py compile_catalog -l $(patsubst chsdi/locale/%/LC_MESSAGES/chsdi.mo,%, $@)
 
 
-DOC_FILES_DEPENDENCIES := $(shell find chsdi/static/doc/source ! -name *.pyc ! -name __pycache__)
+DOC_FILES_DEPENDENCIES := $(shell find chsdi/static/doc/source ! -name "*.pyc" ! -name __pycache__)
 $(DOC_BUILD): $(DOC_FILES_DEPENDENCIES)
 	@echo "${GREEN}Building the documentation...${RESET}";
 	cd chsdi/static/doc && $(SPHINX) -W -b html source build || exit 1 ;
 
 
 guard-%:
 	@ if test "${${*}}" = ""; then \
-	  echo "Environment variable $* not set. Add it to your command."; \
-	  exit 1; \
+		echo "Environment variable $* not set. Add it to your command."; \
+		exit 1; \
 	fi
 
+
+guard-VENV:
```

Comment:
This is a really elegant was of enforcing the setup requirement while not having a need to force a check on every target. I really like it.